tests/fips-test.c

```diff
@@ -706,15 +706,15 @@ void doit(void)
 	}
 	FIPS_POP_CONTEXT(ERROR);
 
-	/* Verify a signature created with ECDSA and SHA-1; approved */
+	/* Verify a signature created with ECDSA and SHA-1; not approved */
 	FIPS_PUSH_CONTEXT();
 	ret = gnutls_pubkey_verify_data2(pubkey, GNUTLS_SIGN_ECDSA_SHA1,
 					 GNUTLS_VERIFY_ALLOW_SIGN_WITH_SHA1, &data,
 					 &ecc256_sha1_sig);
 	if (ret < 0) {
 		fail("gnutls_pubkey_verify_data2 failed\n");
 	}
-	FIPS_POP_CONTEXT(APPROVED);
+	FIPS_POP_CONTEXT(NOT_APPROVED);
```

Comment:
Are we removing sha1?

Reply:
GNUTLS_VERIFY_ALLOW_SIGN_WITH_SHA1 - sign operation only.